### ISSUE TITLE
XSS issue

### ISSUE THREAD

**Issue by adrivanrex** [babel/babel.github.io#1258]
https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&debug=false&code=let%20r%20%3D%20'rex'%3B%0Aalert(document.cookie)%3B

**Comment by xtuc** [comment]
More context about this issue would be welcomed.
Since we allow arbitrary code evaluation and shareable links, this issue is present. I'm not sure we want to change the behavior of the REPL. 

However, we don't store secrets in the cookies on the Babel website.
I would be more worried about HTTP requests used for DDOS for example.

Should we disable somehow request from the REPL?

**Comment by Daniel15** [comment]
Maybe the script should not automatically execute when the page is loaded, and instead wait for the user to click a button? It's fine to automatically transform, just not automatically execute.

**Comment by xtuc** [comment]
Yes, I agree with @Daniel15, I could prompt the user before the execution. He could look at the code and refuse if malware'ish. 

@hzoo what do you think?

**Comment by existentialism** [comment]
Yeah, on load, if evaluate is decoded from the url, we can pop a tooltip on the button that says "click to run" or something.
